CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Logo on the Error page has been fixed - @przspa (#3077)
 - No placeholders / no photos for Get Inspire section in offline - @przspa (#3072)
 - Back icon on product page causing inconsistent behavior - @patzick (#3056)
+- Fixed wrong meta description attribute by page overwrite - @przspa (#3091) 
```

Comment:
We already released rc.1, we need to prepare section for RC.2 and put it there :)

Reply:
@patzick do it like this?

## [1.10.0-rc.2] - 

### Fixed
- Fixed wrong meta description attribute by page overwrite - @przspa (#3091)

Reply:
I'll change it so you can see :)

Reply:
thanks @patzick